next.config.js

```diff
@@ -12,5 +12,17 @@ module.exports = withBundleAnalyzer({
         INTERCOM_COMPANY_NAME: config.get("intercom.company_name"),
         ADMIN_GROUPS: config.get("admin.groups"),
         ADMIN_GROUP_ATTRIBUTE: config.get("admin.group_attribute_name"),
+        IRODS_HOME_PATH: config.get("irods.home_path"),
+        TOOLS_PRIVATE_MAX_CPU_LIMIT: config.has("tools.private.max_cpu_limit")
+            ? config.get("tools.private.max_cpu_limit")
+            : null,
+        TOOLS_PRIVATE_MAX_MEMORY_LIMIT: config.has(
+            "tools.private.max_memory_limit"
+        )
+            ? config.get("tools.private.max_memory_limit")
+            : null,
+        TOOLS_PRIVATE_MAX_DISK_LIMIT: config.has("tools.private.max_disk_limit")
+            ? config.get("tools.private.max_disk_limit")
+            : null,
```

Comment:
I set these to `null` here so that the `ResourceRequirements` component in the app launch form will use its own reasonable defaults:
https://github.com/cyverse-de/sonora/blob/0c60284dba2e9e3c328e6db4b25b09776243f594/src/components/apps/launch/ResourceRequirements.js#L97-L111

I can refactor those defaults so they're set here instead, which is probably better.